handbook/marketing/README.md

```diff
@@ -89,13 +73,11 @@ This is a great option to send when we already have the name, address, and email
 - To create a new contact click the plus sign and fill out the name, email, and shipping address.
 - Select "Configure your item", continue on to "Review", and "Confirm and Send"
 
-##### Send swag for community contributions to Fleet
+**Send swag for community contributions to Fleet**
```

Comment:
Is this one its own responsibility?  If so, it can say "Send swag for a community contribution to Fleet" (singular) and be a top level responsibility, with its own heading.